site/content/entry/citra-progress-report-2017-q4.md

```diff
@@ -0,0 +1,120 @@
++++
+date = "2018-01-21T11:24:00-04:00"
+title = "Citra Progress Report - 2017 Q4"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+As the year draws to a close, work on Citra absolutely doesn't! A lot of technical
```

Comment:
I realize this is 4th quarter, but this is just confusing to users. Can you rephrase this a bit to something like "As we start the new year ... "